cmd/configure/cmd.go

```diff
@@ -78,6 +71,13 @@ func Execute(log logr.Logger, fs afero.Afero, targetDir string) error {
 	for _, containerAttr := range containerAttrs {
 		containerConfigDir := filepath.Join(configFolder, containerAttr.ContainerName)
 
+		err = pmc.Configure(log, fs, inputFolder, targetDir, containerConfigDir, installPath)
```

Comment:
for consistency, why do we call inputFolder and targetDir? can we align here: `inputFolder`, `targetFolder` or
`inputDir`, and `targetDir`?